website/assets/script/sidebar.js

```diff
@@ -0,0 +1,43 @@
+
+// The toggle button
+document.getElementById("sidebarCollapse").addEventListener("click", function(){
+    document.getElementById("sidebar").classList.toggle("active");
+});
+
+let isUlHidden = function (ul) {
+    return !ul.classList.contains('show');
+};
+
+
+// The submenu
+let elements = document.getElementsByClassName('dropdown-toggle');
+for (let i = 0; i < elements.length; i++) {
```

Comment:
for const element of elements

Reply:
I did not know you could write a loop like that.. =)